.grabl/automation.yml

```diff
@@ -64,20 +64,20 @@ build:
         export ARTIFACT_PASSWORD=$REPO_VATICLE_PASSWORD
         bazel run @vaticle_dependencies//distribution/artifact:create-netrc
         .grabl/test-core.sh //tests/behaviour/connection/... --test_output=errors --jobs=1
-    test-behaviour-connection-cluster:
-      image: vaticle-ubuntu-21.04
-      type: foreground
-      command: |
-        pyenv global 3.6.10
-        pip3 install -U pip
-        pip install -r requirements_dev.txt
-        sudo unlink /usr/bin/python3
-        sudo ln -s $(which python3) /usr/bin/python3
-        sudo ln -s /usr/share/pyshared/lsb_release.py /opt/pyenv/versions/3.6.10/lib/python3.6/site-packages/lsb_release.py
-        export ARTIFACT_USERNAME=$REPO_VATICLE_USERNAME
-        export ARTIFACT_PASSWORD=$REPO_VATICLE_PASSWORD
-        bazel run @vaticle_dependencies//distribution/artifact:create-netrc
-        .grabl/test-cluster.sh //tests/behaviour/connection/... --test_output=errors --jobs=1
+#    test-behaviour-connection-cluster:
```

Comment:
Why did we comment these out? Is it just because they are failing? If so, we should keep them enabled and accept that CI jobs fail - it's more dangerous to leave them commented and forget about them

(on the other hand if CI requires them to pass in order to release, then we should temporarily disable them, and immediately bring them back in a follow-up commit)

Reply:
good point, forgot to bring them back!